routes/socket/user-events.js

```diff
@@ -2902,6 +2903,39 @@ module.exports.handleModerationAction = (socket, passport, data, skipCheck, modU
 
 					io.sockets.emit('generalChats', generalChats);
 					break;
+				case 'addbb':
+					Account.findOne({ username: data.userName })
+						.then(acc => {
+							if (acc) {
+								const addbb = new BlanketBanIP({
```

Comment:
It's possible to have multiple BB ips listed for the same account -- is this on purpose? And is it on purpose that this only allows you to mark an account with its LC (and not, say, its signup IP)?